fedora-coreos/provisioning/main.tf

```diff
@@ -11,7 +11,9 @@ terraform {
   }
 }
 
-provider "aws" {}
+provider "aws" {
+  region = "eu-central-1"
+}
```

Comment:
Remove this change, it's already configurable through https://github.com/coreos/coreos-tutorials/commit/e8945b44a80d39910aa4ead9c37424b662908b8c

Reply:
Implemented in https://github.com/coreos/coreos-tutorials/commit/d9ea4e5f3d1cd18f80e1c3334ae7aa4f492af392.